index.html

```diff
@@ -44,5 +44,160 @@ <h2 class="connect-txt">LET'S CONNECT</h2>
+                    no accounts sign-ups required.
+                </p>
+                <ul class="languages">
+                    <li>Html</li>
+                    <li>Css</li>
+                    <li>JavaScript</li>
+                </ul>
+                <button>See Project</button>
+            </div>
+        </div>
+        <div class="project">
+            <img src="./images/project4.png" class="project-img" alt="">
+            <div class="project-txt">
+                <h1 class="project-title">Multi-Post Stories</h1>
+                <div class="caption">
+                    <h3 class="canopy">CANOPY</h3>
+                    <img src="./images/Counter.png" alt="">
+                    <h3>Back end dev</h3>
+                    <img src="./images/Counter.png" alt="">
+                    <h3>2015</h3>
+                </div>
+                <p class="text">
+                    A daily selection of privately personalized reads;
+                    no accounts sign-ups required.
+                </p>
+                <ul class="languages">
+                    <li>Html</li>
+                    <li>Css</li>
+                    <li>JavaScript</li>
+                </ul>
+                <button>See Project</button> 
+            </div>
+        </div>
+    </section>
+    <!-- About section -->
+    <section class="about">
+        <div class="about-content">
+            <h1 class="about-title">About <br> Myself</h1>
+            <p class="text-about">
+                Hello I'm as software developer! I can help you build a 
+                product, feature or website Look through some 
+                of my work and experience! If you like what you 
+                see and have a project you need coded, don't hesitate 
+                to contact me.
+            </p>
+            <div class="connect">
+                <h2 class="connect-txt">LET'S CONNECT</h2>
+                <ul class="icons">
+                    <li class="icon"><img src="./images/twitter.png" alt=""></li>
+                    <li class="icon"><img src="./images/Linkedin-icon.png" alt=""></li>
+                    <li class="icon"><img src="./images/github.png" alt=""></li>
+                    <li class="icon"><img src="./images/angellist.png" alt=""></li>
+                    <li class="icon"><img src="./images/medium.png" alt=""></li>
+                </ul>
+            </div>
+            <button>Get my resume</button>
+            <ul class="knowledge">
+                <li class="scrolled resume">
+                    <div class="knowledge-label scrolled">
+                        <h3>Languages</h3> 
+                        <i class="fa-solid fa-angle-down"></i>
+                    </div>
+                    <div class="langs-label">
+                        <ul>
+                            <li>
+                                <img src="./images/js.png" alt="">
+                                <h3>Javascript</h3>
```

Comment:
- [ ] I believe this is not a heading and it semantically more appropriate to use other tags such as `p` in here.